### PR TITLE
28 add amsterdamumcdb

### PR DESCRIPTION
**Pull request by aegis301**
Added AMDS support. Included the heart rate as first numeric variable.

### CODE REVIEW

File: icu_pipeline/mapper/source/amds/admissions.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            lambda _df: offset_to_period(
+                _df["admissionyeargroup"].str.split("-", expand=True)[
+                    0
+                ],  # only the year group is given, extract the first year as anchor
+                "00:00:00",  # no time is given, so use midnight as anchor
+                _df["admittedat"],
+                _df["dischargedat"],
+            ),
+            axis=1,
+        )
```
Comment (Paul-B98) [inline icu_pipeline/mapper/source/amds/admissions.py:34]: It seems to me the care team is missing for this mapper. If amds isn’t providing any information on this consider using a static value like `ICU`.

```suggestion
    def _to_fhir(self, df: DataFrame) -> DataFrame[FHIREncounter]:
        encounter_df = pd.DataFrame()
        encounter_df[FHIREncounter.subject] = df["patientid"].map(
            lambda id: Reference(reference=str(id), type="AMDS-Patient")
        )
        encounter_df[FHIREncounter.actual_period] = df.apply(
            lambda _df: offset_to_period(
                _df["admissionyeargroup"].str.split("-", expand=True)[
                    0
                ],  # only the year group is given, extract the first year as anchor
                "00:00:00",  # no time is given, so use midnight as anchor
                _df["admittedat"],
                _df["dischargedat"],
            ),
            axis=1,
        )
        encounter_df[FHIREncounter.care_team] = [
            Reference(reference="ICU", type="CareTeam")
        ] * len(df)
        
        return encounter_df.pipe(DataFrame[FHIREncounter])
```

File: test_amds.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+)
+from icu_pipeline.mapper.sink.file import CSVFileSinkMapper
+from icu_pipeline.concept.snomed.observable_entity import (
+    HeartRate,
+)
+
+if __name__ == "__main__":
+    configs = {
+        DataSource.MIMIC: SourceMapperConfiguration(
+            "postgresql+psycopg://christian@localhost/mimiciv_demo"
```
[the remark this answers is not in the document]
Comment (Paul-B98) [reply test_amds.py:15]: Maybe we should store this information in an env-file and load them with pipenv

**Review by Paul-B98 — CHANGES_REQUESTED** [review]

File: icu_pipeline/mapper/source/amds/admissions.py (hunk 1 above)
Comment (aegis301) [reply icu_pipeline/mapper/source/amds/admissions.py:34]: fixed, thank you for pointing out

File: test_amds.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (aegis301) [reply test_amds.py]: done, thank you.

**Review by Paul-B98 — APPROVED** [review]

**Pull request merged**